test/routes/paymentCalculation.integration.test.js

```diff
@@ -1,8 +1,12 @@
-const createServer = require('../../server/createServer')
+let server
 
-describe('POST /parcels/{parcelRef/actions/{actionId}/payment-calculation', () => {
-  let server
+async function setUpServer () {
```

Comment:
We have to require `createServer` after setting up a mock for `data/actions.json` in each test so that mocked data is picked up by the server. This is because the json file is an indirect dependency of `createServer`.

If we imported `createServer` once at the beginning of the tests, then the content of `data/actions.json` would be read immediately and cached so we wouldn't be able to mock its value differently for each test.